test/theme_test.go

```diff
@@ -5,28 +5,92 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 
 	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/theme"
 )
 
-func Test_NewTheme_checkForUniqueColors(t *testing.T) {
-	th := NewTheme().(*configurableTheme)
-	seen := map[string]fyne.ThemeColorName{}
-	for cn, c := range th.colors {
-		r, g, b, a := c.RGBA()
-		key := fmt.Sprintf("%d %d %d %d", r, g, b, a)
-		assert.True(t, seen[key] == "", "color value %#v for color %s already used for color %s in theme %s", c, cn, seen[key], th.name)
-		seen[key] = cn
+// Try to keep these in sync with the existing color names at theme/colors.go.
```

Comment:
The file is called `color.go`. Also, lets add a comment there to update this list once new colours are added.
```suggestion
// Try to keep these in sync with the existing color names at theme/color.go.
```